roles/htcondor/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+# htcondor server setup yaml
+- include_vars: htcondor-public-default-vars.yaml
+- include_vars: htcondor-public-vars.yaml
+
+- import_tasks: create_installation_directories.yaml
+- import_tasks: install_other_required_packages.yaml
+- import_tasks: letsencrypt.yaml
```

Comment:
Should there be conditions when to import?

Reply:
There should be some conditions added for letsencrypt and a couple other imports in the latest update, should I take a look at any others?

Reply:
For now it would be best to remove letsencrypt all together from the condor install.